internal/controller/operator/deployment.go

```diff
@@ -78,6 +82,13 @@ func (r *AuthenticationReconciler) handleDeployments(ctx context.Context, req ct
 		samlConsoleURL = icpConsoleURL
 	}
 
+	auditSecretName, err := r.getAuditSecretNameIfExists(context.TODO(), req.Namespace)
```

Comment:
Please use `ctx` here.
```suggestion
	auditSecretName, err := r.getAuditSecretNameIfExists(ctx, req.Namespace)
```